content/en/docs/armory-admin/kubernetes-account-add.md

```diff
@@ -21,7 +21,7 @@ At a high level, Spinnaker operates in the following way when deploying to Kuber
 * Each Kubernetes account is configured in the `SpinnakerService` manifest under `spec.spinnakerConfig.config.providers.kubernetes.accounts` key if using the Operator, or under the master `.hal/config` YAML file (also known as a "Halconfig"), as an entry in the deploymentConfiguration's `providers.kubernetes.accounts` array if using Halyard.  Each entity has these (and other) fields:
   * `name`: A Spinnaker-internal name used to refer to the Kubernetes account.  This is the item you will select in various Kubernetes stages to indicate that you would like to deploy to this particular Kubernetes account (which again, consists of a Kubernetes cluster/credential/context set)
   * `kubeconfigFile`: A file path referencing the contents of the kubeconfig file for connecting to the target cluster. When using the Operator, this can be any name that should match an entry in `spec.spinnakerConfig.files` where the file contents is copied. If using Halyard, this is a physical file path. This field supports referencing files stored in external [secret engines]({{< ref "secrets" >}})
-  * `onlySpinnakerManaged`: A boolean that indicates whether Spinnaker should only manage / display entities that were created by Spinnaker (i.e., those that have the Spinnaker labels and annotations that indicate that they were created by Spinnaker)
+  * `onlySpinnakerManaged`: When true, Spinnaker will only cache/display applications that have been created by Spinnaker.  Before placing in a "false" state, it is recommended to review the Kubernetes cluster configuration.  Once applied with a false flag, Spinnaker will analyze the cluster and automatically attempt to configure and populate applications for resources already present in Kubernetes, unless limited with `omitNamespaces`. It is important to note the [https://kb.armory.io/s/article/Autogenerated-Applications-Cannot-be-Deleted-w-onlySpinnakerManaged-set-to-false](increased possibilities of zombie applications in MySQL deployments).
```

Comment:
@dwilliams-armory this is the relevant part. Please ignore all the warnings/errors. We just installed a Linter and are working on tuning it. 🙇🏼‍♂️